### PR TITLE
Requested editorial pass on README files

### PR DESCRIPTION
**Pull request by micronAMZN**
Added some formatting and editorial adjustments to README files per request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,239 +2,293 @@
...
-* Many points of extensibility.
 
-Note: There's no penalty for respawn if you are knocked off the level or use the teleporters.
+* 3rd-person character setup
+* Weapons (laser pistols) with a reticle, projectile, and visual FX.
+* A configurable number of rounds (default: 3 rounds)
+* Configurable gem spawning patterns per round to drive player exploration
+* Support for 1 to 15 players
+* Rich sounds and Visual FX support
+* Teleporters to aid player exploration
```
Comment (lmbr-pip) [inline README.md:23]: Nit: Can we:
* Environmental dangers like energy cannons and malfunction shield towers.
* Jump pads to boost players high in the air.

**Review by lmbr-pip — APPROVED** [review]
Only minor concern is that we are repeating a lot of documentation that is covered by o3de.org ie project setup, engine vs project centric

File: README.md [hunk 2]
```diff
@@ -2,239 +2,293 @@
 
 ![MPS SplashScreen](Documentation/Media/splashscreen.png)
 
-A third-person multiplayer game sample for O3DE, where robots battle one another for dominance in a multi-tiered, under-construction, Starbase
+The MultiplayerSample Project is a third-person multiplayer game built on Open 3D Engine (O3DE), where robots battle one another for dominance in a multi-tiered starbase.
```
Comment (mbalfour-amzn) [inline README.md:5]: nit: "under-construction" i believe was meant to describe the condition of the starbase (it's a construction zone), not that the project itself is under construction, so i think it's still a valid descriptive phrase. Maybe "multi-tiered partially-constructed starbase" or "multi-tiered starbase construction site"?

File: README.md [hunk 3]
```diff
@@ -2,239 +2,293 @@
...
-* Move using: `w,a,s,d` 
-* Sprint: Hold `shift`
-* Jump: `space` 
-* Look around: use mouse
-* Fire primary weapon: `left mouse button`
-* See scoreboard: Hold `tab`
-* Open game menu: `esc`
-* Draw/holster active weapon: `e`
+* Move using: **W,A,S,D**
+* Sprint: Hold **Shift**
```
Comment (mbalfour-amzn) [inline README.md:29]: Actually now shift is a toggle, not a hold

**Review by mbalfour-amzn — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (lmbr-pip) [reply README.md:29]: Can probably combine with https://github.com/o3de/o3de-multiplayersample/pull/325

File: README.md [hunk 4]
```diff
@@ -2,239 +2,293 @@
...
+   connect 192.168.0.20
+   ```
+
+2. `launch_server.cfg` contains the initial level to load:
+
+   ```shell
+   LoadLevel Levels/NewStarbase/NewStarbase.spawnable
+   ```
+
+## Step 5. Launch the server
```
Comment (lmbr-pip) [inline README.md:208]: We should probably mention that you need to cd C:\path\to\o3de-multiplayersample\build\windows\bin\profile here. See https://github.com/o3de/o3de-multiplayersample/issues/190

File: README.md (hunk 2 above)
Comment (micronAMZN) [reply README.md:5]: Added "under construction"

File: README.md (hunk 1 above)
Comment (micronAMZN) [reply README.md:23]: Added.

File: README.md (hunk 3 above)
Comment (micronAMZN) [reply README.md:29]: changed to:

* Speed toggle (sprint or walk): Tap **Shift**

File: README.md (hunk 4 above)
Comment (micronAMZN) [reply README.md:208]: Added the path to the command.

**Pull request merged**